src/i18n/de.json

```diff
@@ -53,9 +53,11 @@
     "generator": {
         "reference": "Zeichen",
         "download-pdf": "PDF herunterladen",
+        "download-pdf-again": "Laden Sie das PDF erneut herunter",
```

Comment:
We use "Du" as pronoun in German.

Reply:
You could actually get rid of the pronoun, just write: "PDF erneut herunterladen"